manim/utils/color.py

```diff
@@ -161,6 +164,9 @@ class Colors(Enum):
     red = "#FC6255"
     red_b = "#FF8080"
     red_a = "#F7A1A3"
+    manimce_logo_blue = ("#525893",)
+    manimce_logo_green = ("#87C2A5",)
+    manimce_logo_red = ("#E07A5F",)
```

Comment:
Why tuples?

Reply:
I think ``black`` linting did that. I may have added a ``,`` at the end resulting in the linter making it a tuple? Not sure how that happened, but fixed